src/test/java/com/conveyal/datatools/manager/jobs/NormalizeFieldTransformJobTest.java

```diff
@@ -0,0 +1,136 @@
+package com.conveyal.datatools.manager.jobs;
+
+import com.conveyal.datatools.DatatoolsTest;
+import com.conveyal.datatools.TestUtils;
+import com.conveyal.datatools.manager.models.FeedRetrievalMethod;
+import com.conveyal.datatools.manager.models.FeedSource;
+import com.conveyal.datatools.manager.models.FeedVersion;
+import com.conveyal.datatools.manager.models.Project;
+import com.conveyal.datatools.manager.models.transform.FeedTransformRules;
+import com.conveyal.datatools.manager.models.transform.FeedTransformation;
+import com.conveyal.datatools.manager.models.transform.NormalizeFieldTransformation;
+import com.conveyal.datatools.manager.persistence.Persistence;
+import org.apache.commons.lang3.ArrayUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Date;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import static com.conveyal.datatools.TestUtils.*;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Should be explict import.

Reply:
Thanks, updated in https://github.com/ibi-group/datatools-server/pull/371/commits/ee52bc84d3dd4eb835620d0f05e5215513bfcd44.